src/server-glue/module.ts

```diff
@@ -57,7 +58,7 @@ export const init = async <T extends string>(
 		await cfgLoader.loadConfig(migrator.config);
 
 		const promises: Array<Promise<void>> = [];
-		if (process.env.SBVR_SERVER_ENABLED) {
+		if (optionalVar('SBVR_SERVER_ENABLED')) {
```

Comment:
I think these changes will interfere with the DEFINE plugin: https://github.com/balena-io/pinejs/blob/v14.54.1/build/browser.ts#L18-L24

Reply:
Oh I see...

https://webpack.js.org/plugins/define-plugin/: 
> Note that because the plugin does a direct text replacement, the value given to it must include actual quotes inside of the string itself.

Thus, it's fine for not text-replaced process.env vars... :(